lib/avo/base_resource.rb

```diff
@@ -50,6 +50,7 @@ class BaseResource
     class_attribute :keep_filters_panel_open, default: false
     class_attribute :extra_params
     class_attribute :link_to_child_resource, default: false
+    class_attribute :empty_state_name
```

Comment:
Why empty_state_name? For each language we can have it plural or singular, for example: 
EN: No related comments found.
RO: Nu s-au găsit comentarii relationate. SAU Nici un comentariu relationat gasit.
PT: Nenhum comentário relacionado encontrado.

So it make sense to make it custom. This way some users can use Localization, One, Zero, Other or even a custom string....

Reply:
It seems a bit too much to add a base resource option to translate something. Can't we use the translation helpers? Why doesn't `zero` help? I see that you added `self.empty_state_name = "comments"` somewhere below. Isn't the value of `zero` "comments" in the translation?

Reply:
It is too much, and unnecessary, i don't know why i think it was a good idea while implementing it...